### PR TITLE
Add link to github repo

### PR DESCRIPTION
**Pull request by leosartaj**
@aktech @Sumith1896

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +40,7 @@
             <li><a title='Intro slides' href="http://iamit.in/sympy-pycon/pycon2015.pdf" target="_blank" rel=external>Slides</a></li>
             <li><a title='IPython Notebooks' href="http://nbviewer.ipython.org/github/aktech/sympy-pycon/tree/master/PyCon%20material/tutorial_exercises/" target="_blank" rel=external>IPython Notebooks</a></li>
             <li><a title='HTML Tutorial' href="#" target="_blank" rel=external>HTML Tutorial</a></li>
+            <li><a title='Github Repo' href="http://github.com/aktech/pycon-sympy" target="_blank" rel=external>Source</a></li>
```
Comment (aktech) [inline index.html]: Though, the link works as of now, but the correct link is: https://github.com/aktech/sympy-pycon

**Comment by leosartaj** [comment]
Updated @aktech

**Comment by aktech** [comment]
Thanks! it's in.

**Comment by leosartaj** [comment]
Thanks!

On 15 July 2015 at 17:54, AMiT Kumar notifications@github.com wrote:

> Merged #14 https://github.com/aktech/sympy-pycon/pull/14.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/aktech/sympy-pycon/pull/14#event-356391132.

## 

Regards
_Sartaj Singh_

_Mathematics and Computing_,
Indian Institute of Technology,
Varanasi - 221 005 INDIA

E-mail: singhsartaj94@gmail.com, _sartaj.singh.apm13@itbhu.ac.in
sartaj.singh.apm13@itbhu.ac.in_

**Pull request merged**